src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Config.scala

```
@@ -5,4 +5,5 @@ import com.amazonaws.regions.{ Region, Regions }
 object Config {
   val region = Region.getRegion(Regions.fromName(sys.env.getOrElse("awsRegion", "eu-west-1")))
   val topicArn = sys.env("topicArn")
+  val userTableName = sys.env("userTableName")
```

If for some reason this is not set your Lambda cannot run. So in this case I would fail fast and do something similar to above: 

```
val userTableName = sys.env("userTableName").getOrElse(sys.error("Cannot load userTableName. Lambda will not run."))
```